### ISSUE TITLE
Certbot - ModuleNotFoundError: No module named 'CloudFlare'

### ISSUE THREAD

**Issue by jiriteach** [openappsec/open-appsec-npm#22]
For some reason - this version of NPM does not include the CloudFlare module for Certbot. So if trying to request a SSL cert using Let's Encrypt with a DNS challenge for Cloudflare will error with - 

`ModuleNotFoundError: No module named 'CloudFlare'`

Fix for this is install the module manually - 

`pip install --upgrade cloudflare==2.19.*` and also upgrade - `python3 -m pip install --upgrade pip` on the container. 

Works fine afterwards but would be good to have this fixed at source.

**Comment by orianelou** [comment]
Hi @jiriteach, 

thank you for reporting this and creating a PR, we'll work on this.

**Comment by djr747** [comment]
Hi @orianelou 
I closed the PR since I found an issue based on the static Nginx libs in the repo.  
```
nginx: [emerg] module "/usr/lib/nginx/modules/libngx_module.so" version 1021004 instead of 1025003 in /etc/nginx/nginx.conf:16
```
I was able to get the binaries from the 1.1.17 version of the attachment image to work.  I can create a new PR but I have added GitHub actions for building the image instead of Jenkins from upstream so don't know if you want all that.

**Comment by orianelou** [comment]
Hi @djr747, 

Thank you for your help! this was fixed in our latest release (1.1.20)
